test/e2e/containers/config.go

```diff
@@ -0,0 +1,78 @@
+package containers
+
+// ImageConfig contains all images and their respective tags
+// needed for running e2e tests.
+type ImageConfig struct {
+	InitRepository string
+	InitTag        string
+
+	QuicksilversRepository string
+	QuicksilverTag         string
+
+	RelayerRepository string
+	RelayerTag        string
+}
+
+//nolint:deadcode
+const (
+	// Current Git branch quicksilver repo/version. It is meant to be built locally.
+	// It is used when skipping upgrade by setting QUICKSILVER_E2E_SKIP_UPGRADE to true).
+	// This image should be pre-built with `make docker-build-debug` either in CI or locally.
+
+	CurrentBranchQuickSilverRepository = "quicksilver"
+	CurrentBranchQuicksilverTag        = "debug"
+	// Pre-upgrade quicksilver repo/tag to pull.
+	// It should be uploaded to Docker Hub. QUICKSILVER_E2E_SKIP_UPGRADE should be unset
+	// for this functionality to be used.
+	previousVersionQuicksilverRepository = "quicksilverzone/quicksilver"
+	previousVersionQuicksilverTag        = "v1.4.0-rc9"
+	// Pre-upgrade repo/tag for quicksilver initialization (this should be one version below upgradeVersion)
+	previousVersionInitRepository = "quicksilverzone/quicksilver"
+	previousVersionInitTag        = "v1.2.4"
+	// Hermes repo/version for relayer
+	relayerRepository = "informalsystems/hermes"
+	relayerTag        = "1.2.0"
```

Comment:
we're currently using v1.3.0 in prod

Reply:
https://github.com/ingenuity-build/quicksilver/pull/346/commits/aeea4c1dce28729cc20796db446c0edf5c37693a